### ISSUE TITLE
Browser Usage?

### ISSUE THREAD

**Issue by skipjack** [parshap/node-sanitize-filename#37]
Is this package safe to use in the browser? By the name I was a bit worried (node-sanitize-filename) but it doesn't seem any of the deps are node dependent and, after double checking my webpack bundles, it doesn't seem like library size is an issue (minimized it added only about 2kb to my output). I also read through the code and don't see it using any node built-ins like `path` or `fs`.

Just wanted to check before using in a SPA so I don't run into any surprises. Also, if it's not node dependent, and you don't foresee it being so in the future, maybe it's worth renaming the package to just "sanitize-filename" so other folks don't get tripped up by that? I could see this package being useful in a variety of applications both front-end and back-end.

**Comment by parshap** [comment]
Yes, this package is definitely safe for use in a browser environment! [It even gets tested in browsers!](https://github.com/parshap/node-sanitize-filename/blob/0d21bf13be419fcde5bc3f241672bd29f7e72c63/package.json#L22-L23) The package name is already `sanitize-filename` on npm -- this github repo is just prefixed with `node-`.

A note in the readme about browser usage would be good to have.

**Comment by skipjack** [comment]
@parshap thanks for the response. Yeah, it was just the repo name that threw me off. Seeing as changing the repo name wouldn't be much of a breaking change, you may want to consider that. But yeah a note in the readme would be nice as well.

**Comment by jslick** [comment]
When I try to use it in browser, I get the error 

> ReferenceError:  Buffer is not defined

But, Buffer is a node library.
This is coming from the dependency truncate-utf8-bytes (index.js):
`var getLength = Buffer.byteLength.bind(Buffer);`
( https://github.com/parshap/truncate-utf8-bytes/blob/c8fcebc8be093c8bd8db1e7d75c09b9fce7e4708/index.js#L4 )

FWIW, I'm using rollup w/ the resolve and CJS plugins:
```
            plugins:  [
                resolve({
                    jsnext: true,
                    main:   true,
                }),

                commonjs({
                    include:  'node_modules/**',
                }),

                // others...
            ]
```

**Comment by parshap** [comment]
@jslick: You'll need some implementation of the node core modules. [Browserify](https://github.com/substack/node-browserify#compatibility) and [Webpack](https://github.com/webpack/node-libs-browser) do this, I'm not sure what the right solution for Rollup is.

**Comment by jslick** [comment]
Thanks parshap.  I found the solution.  It turns out, truncate-utf8-bytes specifies pkg.browser, which specifies a file that omits Node dependencies.  I just had to add `browser: true` to my rollup resolve plugin.
